docs/quickstart.md

```diff
@@ -25,12 +25,12 @@ pip install -U rapidata
 
 Orders are managed through the [`RapidataClient`](reference/rapidata/rapidata_client/rapidata_client.md#rapidata.rapidata_client.rapidata_client.RapidataClient).
 
-Create a client as follows, this will save your credentials in a local file so you don't have to log in again on that machine:
+Create a client as follows, this will save your credentials in your `.config/rapidata/credentials.json` file so you don't have to log in again on that machine:
```

Comment:
```suggestion
Create a client as follows, this will save your credentials in your `~/.config/rapidata/credentials.json` file so you don't have to log in again on that machine:
```

Not sure if my suggestion is valid on Windows